gulpfile.js

```diff
@@ -17,7 +17,6 @@ var filter = require('gulp-filter');
 var buffer = require('vinyl-buffer');
 var source = require('vinyl-source-stream');
 var rename = require('gulp-rename');
-var size = require('gulp-size');
```

Comment:
The root of the Error of the problems has been removed, along with all references to it below.